### ISSUE TITLE
Loading never end

### ISSUE THREAD

**Issue by gcn59** [jaszhix/NoMansConnect#176]
- Description of the problem
Loading keeps going
- Windows version
Latest one 
- NMC version
1.10.1
- Absolute path to your storage_x.hg files.
I don't know
- Absolute path to your NMS install directory.
I don't know

- NMC.log : https://pastebin.com/pqmYRV8R
- settings.json : https://pastebin.com/kRCH3rzt

- How can this bug be reproduced?
Start NoMansConnect

**Comment by jaszhix** [comment]
It's trying to read a save files in `C:\Users\Shadow\AppData\Roaming\HelloGames\NMS` - does that directory exist? If so, does your user account have read access to it?

**Comment by gcn59** [comment]
Yes, that directory exist, and yes my user account have the right to read and write to it.
Also, if it can help, I'm using [Shadow](https://shadow.tech), a cloud powered computer which is a virtual machine. Does it block Noman's connect ?

**Comment by jaszhix** [comment]
I'm not familiar with Shadow, but it does seem outside of the normal parameters of the way the app was tested. I would try to setup NMC locally on your machine first so it can do an initial auth handshake with the server, and then it should work through cloud providers.

**Comment by gcn59** [comment]
Sorry, but I think you do not understand. (I'm friendly, just English isn't my native language)
My local machine has nothing shared with Shadow, because Shadow is a whole computer in the cloud. Even if I download Noman's connect on my machine, it wouldn't work very well, knowing my local machine is literally a potato in term of power. 
I would try it, but, can you suggest something else ?
